packages/core/src/connectivity/scp-cf/xsuaa-service.ts

```diff
@@ -204,23 +204,37 @@ export function fetchVerificationKeys(
       logger.warn(
         'JKU field from the JWT not provided. Use xsuaaClient.url/token_keys as fallback. ' +
           'This will not work for subscriber accounts created after 14th of April 2020.' +
-          'Please provide the right URL given by the field JKU present in the JWT header.'
+          'Please provide the right URL given by the field `jku` in the JWT header.'
+      );
+      return executeFetchVerificationKeys(
+        `${xsuaaUriOrCredentials.url}/token_keys`,
+        xsuaaUriOrCredentials.clientid,
+        xsuaaUriOrCredentials.clientsecret
```

Comment:
I thought we discussed that the credentials are not needed to fethc the verification key.

Reply:
Ah, this is the same URL, right? Or can it be something different?

Reply:
I do not understand the question.

Reply:
Clarified in person, behavior should be the same for old xsuaa as well.